src/code/code_800FC620.c

```diff
@@ -20,14 +20,14 @@ char D_80134488[0x18] = {
     0xFF, 0x80, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x80, 0x00, 0x00, 0x00,
 };
 
-s32 Overlay_Load(u32 vRomStart, u32 vRomEnd, void* vRamStart, void* vRamEnd, void* allocatedVRamAddr) {
+s32 Overlay_Load(uintptr_t vRomStart, uintptr_t vRomEnd, void* vRamStart, void* vRamEnd, void* allocatedVRamAddr) {
```

Comment:
This function is so ultra-specific (pun intended) that it seems rather fruitless to attempt to make it more generally-applicable: it only works for 32-bit MIPS relocs, and not even all of those.

Reply:
I think its still good to have these types for documentation reasons.  It makes it much clearer that they should be treated as pointers.